### DISCUSSION TITLE
Combined/Joint Letters

### DISCUSSION THREAD

**Discussion by ttkdisc** [florisboard/florisboard#1833] [Q&A] [answered]
I am trying to add Devnagari Hindi Compact Layout. There are some joint letters like:

क्ष which is combination of  &#2325 &#2381 &#2359

What Should i put in code for this letter?

**Comment by ttkdisc** [comment]
going to use -255 in code .Since it is used in .com , .gov , .edu and similar stuffs which are also combined letters.

**Comment by patrickgold** [comment, marked as answer]
Sorry for answering this late, have a lot of stuff going on. FlorisBoard has a special key for such instances (replace `?` with the actual key label):

```json
{ "$": "multi_text_key", "codePoints": [2325, 2381, 2359], "label": "?" }
```

Should you need popups too just add them after the label, works the same as for a normal key.

This has never been used by any layout in practice though so your layout would be the first to make use of this neat little feature :)

**Reply by ttkdisc** [reply]
Thanks for answer and for creating such beautiful keyboard :)
Modified my files and will send PR from other account.
